samples/bluetooth/mesh/dfu/distributor/sample.yaml

```diff
@@ -32,14 +32,26 @@ tests:
       - bluetooth
       - ci_build
       - sysbuild
-  sample.bluetooth.mesh_dfu_distributor.ext_flash:
+  sample.bluetooth.mesh_dfu_distributor.dfu_ext_flash:
     sysbuild: true
     build_only: true
     integration_platforms:
       - nrf52840dk/nrf52840
```

Comment:
I guess adding here nrf54l was forgotten. Hence this is out of scope of this PR.

Reply:
Nice, thanks. If it somehow starts failing Twister because of this just remove it again, I'll handle it in that case.